### PR TITLE
add new localization

### PR DESCRIPTION
**Pull request by sojalei**
need testing,
P:S I lost my phone and now i can't open telegram

### CODE REVIEW

File: lang/irc.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "en": {
     "help": "Equivalent of IRC /me command"
+  },
+  "ru": {
+    "help" : "Эквивалент Ирки /me command"
```
Comment (evgfilim1) [inline lang/irc.json]: > /me command

English words in Russian strings... Fix it

File: lang/kickban.json [hunk 2]
```diff
@@ -5,5 +5,19 @@
     "kicked": "{0} kicked {1}",
     "banned": "{0} banned {1}",
     "unbanned": "{0} unbanned {1}"
+  },
+  "ru": {
+    "help" : "Этот модуль помогает администраторам групп\nДоступные комманды:\n/kick - кикнуть пользователя\n/ban - забанить пользователя\n/unban - разбанить пользователя",
+    "not_admin": "У тебя нет прав!",
+    "kicked": "{0} кикнутые {1}",
```
Comment (evgfilim1) [inline lang/kickban.json]: I think there should be "кикнул(-а)" and so on

File: lang/ping.json [hunk 3]
```diff
@@ -2,5 +2,13 @@
   "en": {
     "help": "This module just replies \"Pong!\" to /ping command",
     "pong": "Pong!"
+  },
+  "ru": {
+    "help": "Этот модуль отвечает \"Pong!\" на /ping command",
```
Comment (evgfilim1) [inline lang/ping.json]: > /ping command

See first comment

Also, "Pong!" here, but "Понг!" below. I think it should be fixed

File: lang/whoami.json [hunk 4]
```diff
@@ -4,5 +4,17 @@
     "info_common": "You are {0}\nYour ID: {1}\n",
     "info_reply": "Replied to {0}\nReplied to ID: {1}\n",
     "info_forward": "Forwarded from {0}\nForwarded from ID: {1}"
+  },
+  "ru": {
+    "help": "Этот модуль отвечает на /whoami информацией о пользователе\n\nАльтернативы: /who, /whois",
+    "info_common": "Вы {0}\nВаш АйДи: {1}\n",
+    "info_reply": "Отвечено на {0}\nОтвечено на АйДи: {1}\n",
+    "info_forward": "Отослано с {0}\nОтослано с АйДи: {1}"
```
Comment (evgfilim1) [inline lang/whoami.json]: I think there should be "Переслано с ..."

File: lang/msgdel.json [hunk 5]
```diff
@@ -1,5 +1,11 @@
 {
   "en": {
     "help": "This module helps to delete any message sent by bot\nUsage:\nReply to message which you want to delete and type /delete"
+  },
+  "ru": {
+    "help": "Этот модуль удаляет сообщения отправленные ботом\nИспользовать:\nОтветить на сообщение которое вы хотите удалить и написать /delete"
```
Comment (evgfilim1) [inline lang/msgdel.json]: Fix it: 
- "сообщения`,` отправленные"
- "сообщение`,` которое ...`,` и написать"

**Review by evgfilim1 — CHANGES_REQUESTED** [review]

**Comment by sojalei** [comment]
check

File: lang/irc.json [hunk 6]
```diff
@@ -1,5 +1,11 @@
 {
   "en": {
     "help": "Equivalent of IRC /me command"
+  },
+  "ru": {
+    "help" : "Эквивалент Ирки /me комманда"
```
Comment (evgfilim1) [inline lang/irc.json]: I think this should be "Эквивалент команды /me в Ирке" (also fix translation below)

File: lang/ping.json [hunk 7]
```diff
@@ -2,5 +2,13 @@
   "en": {
     "help": "This module just replies \"Pong!\" to /ping command",
     "pong": "Pong!"
+  },
+  "ru": {
+    "help": "Этот модуль отвечает \"Понг!\" на /ping комманда",
```
Comment (evgfilim1) [inline lang/ping.json]: "... на команду /ping", I think

File: lang/msgdel.json [hunk 8]
```diff
@@ -1,5 +1,11 @@
 {
   "en": {
     "help": "This module helps to delete any message sent by bot\nUsage:\nReply to message which you want to delete and type /delete"
+  },
+  "ru": {
+    "help": "Этот модуль удаляет сообщения, отправленные ботом\nИспользовать:\nОтветить на сообщение, которое вы хотите удалить, и написать /delete"
```
Comment (evgfilim1) [inline lang/msgdel.json]: Put "Как использовать:" or "Доступные команды:" instead

**Review by evgfilim1 — CHANGES_REQUESTED** [review]
Last thing before merging

**Review by evgfilim1 — APPROVED** [review]

**Pull request merged**